src/olympia/blocklist/tests/test_models.py

```diff
@@ -42,6 +43,23 @@ def test_is_imported_from_kinto_regex(self):
         block.update(kinto_id='1234567890')
         assert not block.is_imported_from_kinto_regex
 
+    def is_readonly(self):
+        block = Block.objects.create(guid='foo@baa', updated_by=user_factory())
+        # not read only by default
+        assert not block.is_readonly
+        # but should be if there's an active BlocklistSubmission
+        block.active_submissions = [object()]  # just needs to be non-empty
+        assert block.is_readonly
+
+        # otherwise kinto_id being non-false means it's imported, so readonly
+        del block.active_submissions
```

Comment:
It would probably read better with an assignment to an empty list:
```suggestion
        block.active_submissions = []
```

Reply:
deleting it resets the cached property so the query runs - I want to check the query works too.

Reply:
ah ok, then maybe add a comment? I tried with empty list and the test passed too.